service/publishing.gradle

```diff
@@ -28,6 +28,7 @@ jib {
         paths = [file(jibExtraDirectory)]
     }
     container {
+        creationTime = 'USE_CURRENT_TIMESTAMP'
```

Comment:
I think we should go the other direction, and also remove `filesModificationTime` from the config, so that jib builds are reproducible. https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#why-is-my-image-created-48-years-ago https://reproducible-builds.org

Reply:
happy to remove it ... I read that link, too, and found it weirdly bizarre all of our docker images from this build process were considered "built at epoch" (read 'decades ago')

Reply:
... as for `filesModificationTime` it is specifically called out that it is there to prevent ui caching. maybe we should be wiring in ... something else? githsha?